src/AspNet.Security.OAuth.Vkontakte/VkontakteAuthenticationHandler.cs

```diff
@@ -50,6 +51,12 @@ public partial class VkontakteAuthenticationHandler : OAuthHandler<VkontakteAuth
         }
 
         using var container = JsonDocument.Parse(await response.Content.ReadAsStringAsync(Context.RequestAborted));
+
+        if (container.RootElement.TryGetProperty("error", out _))
+        {
+            throw new AuthenticationException();
```

Comment:
What does the error look like? Can you use the contents to build a meaningful exception message?

Reply:
ok, no problem